CMakeLists.txt

```diff
@@ -31,6 +31,7 @@
 ############################################################
 
 cmake_minimum_required(VERSION 3.21)
+include("cmake/global-policies.cmake" NO_POLICY_SCOPE)
```

Comment:
Don't need either the path prefix or the extension for `include`. Don't want them either, since they fix the code structure of CMake modules.

Reply:
I moved the module path setup earlier in the file to accommodate this.